[Term Entry] Python Random function: .triangular() (#5055)
Codecademy/docs: content/python/concepts/random-module/terms/triangular/triangular.md

> ---
> Title: 'triangular'
> Description: 'Returns a random floating number from a triangular distribution.'
> Subjects: 
>   -"Computer Science"
>   -"Data Science"
> Tags:
>   -"Functions"
>   -"Methods"
>   -"Random"
> Catalog Content: 
>   -"learn-python-3"
>   -"paths/computer-science"
>   -"paths/data-science"
> ---

Reviewer: ```suggestion
---
Title: '.triangular()'
Description: 'Returns a random floating number from a triangular distribution.'
Subjects: 
  -'Computer Science'
  -'Data Science'
Tags:
  -'Functions'
  -'Methods'
  -'Random'
Catalog Content: 
  -'learn-python-3'
  -'paths/computer-science'
  -"paths/data-science'
---
```

Reviewer: Meta data to be written in single quotes ( ' ' ) and the title should be the term itself.

> "The ** `.triangular()` ** method returns a random floating number from a triangular distribution. Input parameters are `low`, `high` and `mode`. If `mode` not provided, the midpoint is set to default."

Reviewer: ```suggestion
The **`.triangular()`** method returns a random floating-point number from a triangular distribution. The input parameters are `low`, `high`, and `mode`. If `mode` is not provided, it defaults to the midpoint between `low` and `high`.
```

## Syntax

```pseudo

> random.triangular(low,high,mode)

Reviewer: ```suggestion
random.triangular(low, high, mode)
```

```

> Where `low` and `high` correspond to the required lower and upper limits of distribution, and `mode` is an optional input for peak of the distribution.

Reviewer: Instead of this, could you define the parameters in the following format:

- `low`: description
- `high`: description
- `mode`: description

## Example

In the example below, `.triangular()` is used to return a random integer between 50 and 500, biased towards 400.

```py
import random

> print(random.triangular(50,500,400)

Reviewer: ```suggestion
print(random.triangular(50,500,400))
```

Reviewer: missing `)`

```

> Example output:

Reviewer: ```suggestion
The code above produces the following output:

```

```shell
395
```

> 

Reviewer: We can add a note here, after the output, saying that it will always generate a random value.

## Codebyte Example

> The following examples prints a random floating-point value between 1 and 100, biased towards 70; 350 and 1000, biased towards 385; 4 and 5, biased towards 4; 1 and 100 with default bias at midpoint.

Reviewer: ```suggestion
Run the following code to see how random.triangular() generates random values with different biases:
```

```codebyte/python

> import random
> 
> print('random.triangular(1,100,80)
> print('random.triangular(350,1000,385)
> print('random.triangular(4,5,4)
> print('random.triangular(1,100)

Reviewer: ```suggestion
import random

print(random.triangular(1,100,80))
print(random.triangular(350,1000,385))
print(random.triangular(4,5,4))
print(random.triangular(1,100))
```
